crates/l2/contracts/README.md

```diff
@@ -0,0 +1,12 @@
+# Ethrex L2 contracts
+
+There are two L1 contracts: OnChainProposer and CommonBridge. Both contracts are deployed using UUPS proxies, so they are upgradeables.
+
+### Upgrade the contracts
```

Comment:
Can we expand this a little bit with some `rex` commands step-by-step? Something like:

```
rex deploy <NEW_CONTRACT_BYTECODE> <PRIVATE_KEY>
rex send <DEPLOYED_CONTRACT_ADDRESS> 0 <PRIVATE_KEY> -- "upgradeToAndCall(address newImplementation, bytes memory data)" 0 <NEW_IMPLEMENTATION_ADDRESS> <DATA_BYTES>
```

Reply:
Done in [0d87556](https://github.com/lambdaclass/ethrex/pull/2660/commits/0d875564cd6d64b5647564710630ffbb27024ba7) and [f9b561c](https://github.com/lambdaclass/ethrex/pull/2660/commits/f9b561c6cab0be5babf826f9ba75ee79a8eff6e7)